hydrator-common/src/main/java/co/cask/hydrator/common/KeyValueListParser.java

```diff
@@ -82,11 +82,15 @@ public boolean hasNext() {
       Iterator<String> keyValIter = Splitter.on(keyValDelimiter).trimResults().split(pair).iterator();
       String key = keyValIter.next();
       if (!keyValIter.hasNext()) {
-        throw new IllegalArgumentException("Invalid syntax for key-value pair in list: " + pair);
+        throw new IllegalArgumentException(String.format("Invalid syntax for key-value pair in list: %s. " +
```

Comment:
don't think we need to put the pair in the message twice, right after each other.

Invalid syntax for key-value pair: %s. It is expected to be...